data/human/variants.txt

```diff
@@ -1123,7 +1123,7 @@ ship "Fury" "Fury (Flamethrower)"
 ship "Fury" "Fury (Gatling)"
 	outfits
 		"Gatling Gun" 4
-		"Gatling Gun Ammo" 6000
+		"Gatling Gun Ammo" 12000
```

Comment:
This one was also [intentional](https://github.com/endless-sky/endless-sky/pull/1296#issuecomment-230366859).

Reply:
Got it! Thanks, I reverted that change in this PR